exampleSite/content/docs/customization.md

```diff
@@ -179,6 +179,20 @@ your site's root. The `icon` key filed, which is optional, should be a path rela
   icon = "youtube.svg"
 ```
 
+You can also specify the `rel` attribute for the link. By default, the attribute value is `"noopener noreferrer"`. You can remove the attribute completely by setting its value to `false`.
```

Comment:
Good, but in the wrong place. Now your additions split the paragraphs about SVG icons. You need to move your part upwards or downwards.

Reply:
Oops, sorry about that. I amended the commit